src/Components/BookPage/AboutBook.js

```diff
@@ -0,0 +1,318 @@
+
+  // Process date_published
+  const publishDate = processDate(datePublished);
+
+  const authorLinks = authors.map((author, i) => {
+    // Add comment to links unless it is the last
+    const link = (i !== authors.length - 1) ? `${author.name}, ` : author.name;
+    return (
+      <span key={author.id}>
+        <Link style={styles.link} to={`/creator/${author.id}`}>
+          {link}
+        </Link>
+      </span>
+    );
+  });
+
+  const illustratorLinks = illustrators.map((illustrator, i) => {
+    const link = (i !== illustrators.length - 1) ? `${illustrator.name}, ` : illustrator.name;
+    return (
+      <span key={illustrator.id}>
+        <Link style={styles.link} to={`/creator/${illustrator.id}`}>
+          {link}
+        </Link>
+      </span>
+    );
+  });
+  return (
+    <Card sx={styles.sideCard}>
+      <CardContent>
+        <Box sx={styles.sideCardContainer}>
+          <Typography sx={styles.sideCardTitle}>About This Book</Typography>
+          <Box sx={styles.sideCardLinkContainer}>
+            <Typography sx={styles.creator}> Written by:&nbsp;&nbsp;</Typography>
+            {authorLinks}
+            <br />
+            <Typography sx={styles.creator}> Illustrated by:&nbsp;&nbsp;</Typography>
+            {illustratorLinks}
+          </Box>
+          <div style={styles.blockContainer}>
+            <div style={styles.block}>
+              <Typography sx={styles.bolded}>
+                {ageRange}
+              </Typography>
+              <p style={styles.sub}>Age Range</p>
+            </div>
+            {
+                    (gradeMin || gradeMax) ? (
+                      <div style={styles.block}>
+                        <Box sx={styles.bolded}>
+                          {gradeRange}
+                          {' '}
+                        </Box>
+                        <p style={styles.sub}>Reading Level</p>
+                      </div>
+                    ) : <div style={styles.block} />
+                }
+            {
+                  (bookType) ? (
+                    <div style={styles.block}>
+                      <Box sx={styles.bolded}>
+                        {bookType}
+                        {' '}
+                      </Box>
+                      <p style={styles.sub}>Book Format</p>
+                    </div>
+                  ) : <div style={styles.block} />
+                }
+            {
+                  (datePublished) ? (
+                    <div style={styles.block}>
+                      <Box sx={styles.bolded}>
+                        {`${publishDate[1]} ${publishDate[2]}, ${publishDate[3]}`}
+                        {' '}
+                      </Box>
+                      <p style={styles.sub}>Date Published</p>
+                    </div>
+                  ) : <div style={styles.block} />
+                }
+          </div>
+          {
+                (bookshopURL) && (
+                  <Button
```

Comment:
I think you need to add a specific style prop to buttons so that they don't automatically capitalize the label!